packages/react/src/Prose/Prose.module.css

```diff
@@ -140,7 +140,7 @@
 
 .Prose ul {
   list-style-type: image;
-  list-style-image: url("data:image/svg+xml;charset=UTF-8,%3csvg xmlns='http://www.w3.org/2000/svg' aria-hidden='true' focusable='false' role='img' viewBox='0 -3 16 16' width='16' height='16' fill='currentColor' style='display: inline-block; vertical-align: text-bottom; overflow: visible;'%3e%3cpath d='M2 7.75A.75.75 0 0 1 2.75 7h10a.75.75 0 0 1 0 1.5h-10A.75.75 0 0 1 2 7.75Z'%3e%3c/path%3e%3c/svg%3e ");
+  list-style-image: var(--brand-Prose-imageUrl-default);
```

Comment:
This token doesn't mention lists 🤔, and I can't find the previous thread for the previous suggestion.. Could we be a little more specific please?

Could you change this to `--brand-Prose-unorderedList-imageUrl` please. `default` could be interesting for the future, if we add more styles, but perhaps a little redundant in the meantime.